Missing include, probably due to jgroff-0.99 being based on groff-1.9
initially.
$OpenBSD$
--- hpftodit/hpftodit.cc.orig	Tue Jun 27 14:59:39 1995
+++ hpftodit/hpftodit.cc	Sun May  9 22:33:01 2010
@@ -34,6 +34,7 @@ put filename in error messages (or fix lib)
 #include <string.h>
 #include <errno.h>
 #include "assert.h"
+#include "eucmac.h"
 #include "lib.h"
 #include "posix.h"
 #include "errarg.h"
